src/process/tags.md

```diff
@@ -0,0 +1,56 @@
+# Process tags
+
+Description of tags, used for projects.
+
+## Status
+- refine `#c377e0` - task already have resolving code, but code needs some fixes.
+- take it `#0052cc` - any teammate can take this task and start working with it.
+- discuss `#fbca04` - task needs to finish discussion before start working.
+- needs info `#c2e0c6` - task needs additional requirements to start working.
+- blocked `#f8c700`
+- blocker `#cc0701`
+
+## Niceness
+- easy `#fef2c0` - new teammates can start with such tasks to become familiar with project.
+- hard `#aea270` - task requires strong skills in arch, management or project domain area.
+
+## Type
+This tags are mutually exclusive.
+
+- bug `#ee0701`
+- feature `#84b6eb` - task provides some new functionality for project users.
+- cleanup `#bfd4f2` - everything that do project stronger, flexible, reusable.
```

Comment:
maybe be better use more common in github tag `enhancement` instead of `cleanup`

Reply:
@artemiy312 , i tried to find the answer in fidals and 8gen teams what `enhancement` means.
But it seems noone knows what it means. It's too common term. For example yesterday i just turned `enhancement` in a `feature` tag in 8gen project. And for 90% tasks with tag `enhancement` it was right.

I chose "cleanup" term for two reasons:
- it was the best term from my head
- ... and lately i found that django [uses cleanup term too](https://docs.djangoproject.com/en/2.0/internals/contributing/triaging-tickets/#type)